connections/ws.js

```diff
@@ -19,7 +19,11 @@ class WebSocketConnection extends EventEmitter {
 
   create (url, options) {
     if (!WebSocket) this.emit('error', new Error('No WebSocket transport available'))
-    try { this.socket = new WebSocket(url, [], { origin: options.origin }) } catch (e) { return this.emit('error', e) }
+    try {
+      this.socket = new WebSocket(url, [], { origin: options.origin, rejectUnauthorized: false })
```

Comment:
Same here, this is required by our use of self-signed certs, they don't work otherwise. However I missed a bit, this should apply only for `wss`, same as for `https` above.

Reply:
for `wss` I got it but not for `https`